### PR TITLE
Update CRDs

### PR DESCRIPTION
**Pull request by danudey**
Update CRDs from enterprise

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Review by rene-dekker — APPROVED** [review]
lgtm

**Pull request merged**
